sys/shell/commands/sc_gnrc_rpl.c

```diff
@@ -174,9 +174,9 @@ int _gnrc_rpl_trickle_start(char *arg1)
     return 0;
 }
 
-int _gnrc_rpl_send_dis(void)
+int _gnrc_rpl_send_dis(ipv6_addr_t *addr, uint8_t flags)
 {
-    gnrc_rpl_send_DIS(NULL, (ipv6_addr_t *) &ipv6_addr_all_rpl_nodes);
+    gnrc_rpl_send_DIS(NULL, (ipv6_addr_t *) &ipv6_addr_all_rpl_nodes, flags);
```

Comment:
shouldn't this be `gnrc_rpl_send_DIS(NULL, addr, flags)`? otherwise parameter `addr` is unused here